### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by Jia-A**
In this PR, I have added following features for Shoonya 
 - Create playlist
 - Add videos to customized playlist.
 - Delete complete playlist.
 - Delete a video from a particular playlist.
 - Add to likes, watch later from playlist videos.

### CODE REVIEW

**Review by devananth — APPROVED** [review]
- Code looks well structured and clean
- Good work !

File: src/styles/playlists.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.relative {
+    position: relative;
+}
+
+.no-playlist{
+    color: #a0b2b9;
```
Comment (mehulsatardekar) [inline src/styles/playlists.css:6]: I see that this color value is being used twice (line no 6 and 16) in this file so use the color variable

File: src/pages/playlists.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { Navbar } from "../components/navbar";
+import { Sidebar } from "../components/sidebar";
+import { Link } from "react-router-dom";
+import "../styles/homepage.css"
+import "../root.css";
+import "../styles/playlists.css"
+import { useState } from "react";
+import { useVideo } from "../context/videoContext";
```
Comment (mehulsatardekar) [inline src/pages/playlists.jsx:8]: this can be refactor (use barrel export) for importing files

File: src/pages/playlists.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { Navbar } from "../components/navbar";
+import { Sidebar } from "../components/sidebar";
```
Comment (mehulsatardekar) [inline src/pages/playlists.jsx:2]: use Barrel export here

**Review by mehulsatardekar — APPROVED** [review]
Suggestion: use barrel export for importing files and rest  things looks good

File: src/styles/playlists.css (hunk 1 above)
Comment (Jia-A) [reply src/styles/playlists.css:6]: Okay.

**Pull request merged**